src/main/java/io/srqsoftware/DefaultBadgeService.java

```diff
@@ -54,6 +54,19 @@ public Badge getBadge(String badgeId) {
 		return jdbcTemplate.queryForObject(query, new String[] {badgeId}, new BadgeRowMapper());
 	}
 
+	@Override
+	public boolean initAdmin(String pw) {
```

Comment:
could let the exception bubble up the stack and let the caller handle it like we do in other controllers, would avoid the boolean and would be good to log the stack trace so we know why it failed